CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# How to contribute to AMF
+
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+The following is a set of guidelines for contributing to AMF:
+
+## Did you find a bug?
+- Ensure the bug was not already reported by searching on GitHub under Issues.
+- If you're unable to find an open issue addressing the problem, open a new one. If possible, use the relevant bug report templates to create the issue.
+If not, be sure to include a title and clear description, as much relevant information as possible, 
+and a code sample or an executable test case demonstrating the expected 
+behavior that is not occurring.
+
+## Contributing Changes
+- Read and sign the [Contributors License Agreement](#contributors-license-agreement)
+- Open a new GitHub pull request with the change.
+- Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+- Before submitting, please read the [Code contributions](#code-contributions) section to know more about the technical contribution requirements.
+
+## Code Contributions
+
+### Development Requirements
+* Node
+* Ruby
+
+### How to update documentation
+
+To update documentation you must edit the `example.md` files you want to change. If you've added a new subtitle, it should be added to the left index by adding a list element inside the `_includes/` folder in the corresponding `example_menu.html` file:
+
+```html
+  <li><a href="#example subtitle">Example Subtitle</a></li>
+```
+
+GitHub Pages builds the files in the `master` branch. Every change made (i.e. merged PR) updates the public documentation almost instantly.
```

Comment:
What about the gh-pages branch? What is it for?

Reply:
there's none